packages/ember-htmlbars/lib/keywords/get.js

```diff
@@ -35,8 +35,10 @@ let DynamicKeyStream = BasicStream.extend({
 
   key() {
     let key = this.keyDep.getValue();
-    if (typeof key === 'string') {
+    if (typeof key === 'symbol') {
```

Comment:
Thanks for the help. However, it seems there isn't support for symbols in templates. Do we want to add symbol support for the `{{get}}` keyword?

For example, something like this doesn't work either:
```js
['@test should be able to get an object value with numeric keys']() {
  let first = Symbol('first');
  let second = Symbol('second');
  let third = Symbol('third');

  this.render(`{{#each indexes as |index|}}[{{get items index}}]{{/each}}`, {
    indexes: [first, second, third],
    items: {
      [first]: 'First',
      [second]: 'Second',
      [third]: 'Third'
    }
  });

°°°
```